synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -149,9 +150,20 @@ export class SynopsysToolsParameter {
       }
     }
 
-    const isPullRequest = isPullRequestEvent();
+    const azureData = this.getAzureRepoInfo();
```

Comment:
why do we need this in polaris flow, we don't need to fetch all this information, all you need is pull request id, so better it can be fetched here itself.

Reply:
But to fetch the pull request id, the API we call need these `azureData` and using this `azureData` the API url is formed.